_posts/2013-01-11-navigate-using-just-your-keyboard.md

```diff
@@ -9,11 +9,25 @@ categories:
   - Quick Tests
 ---
 
-One quick way that you can check your accessibility is to navigate through the site using nothing but your keyboard.
+Testing your site for keyboard navigation will help ensure that keyboard users can access your site.
 
-1. Un-plug/turn off your mouse.
-2. Use the <kbd>Tab</kbd> key browse.
-3. Use <kbd>Enter</kbd> to select elements.
-4. Some menus and form elements may require <kbd>&uarr;</kbd> <kbd>&darr;</kbd> <kbd>&larr;</kbd> <kbd>&rarr;</kbd> arrow keys.
+As a bonus, sites that can be navigated with a keyboard are more likely to be navigable with other input devices such as switches.
 
-This is by no means a complete accessibility check, but it can quickly help you find major accessibility problems.
+## How to navigate a website with a keyboard
+Generally, pressing <kbd>tab</kbd> moves focus between interactive elements and pressing <kbd>enter</kbd> will interact with them.
+
+If you are using Safari, <kbd>option-tab</kbd> is the default keyboard shortcut to move focus between elements. You can update your settings to make <kbd>tab</kbd> the default navigation key by following the instructions in this article: [Quick Tip: Safari Keyboard Navigation](https://a11yproject.com/posts/safari-keyboard-navigation/)
+
+Additional keyboard shortcuts are described here: [WebAIM - Keyboard Accessibility](https://webaim.org/techniques/keyboard/)
+
+## Things to keep in mind while testing
+It is not enough to simply tab through a webpage from start to end. While navigating the site, keep these things in mind:
+- Interactive elements should be functional using only keyboard input.
+- The focused element should be visible and have an obvious focus style. See article: ["Quick Tip: Never remove CSS outlines"](https://a11yproject.com/posts/never-remove-css-outlines/).
+- Focus should move between elements as they appear on the page and not jump back and forth.
+- There should be a way to skip past lengthy spans of content, like navigation menus. See article: ["How–to: Use Skip Navigation links"](https://a11yproject.com/posts/skip-nav-links/).
+
+### Further Reading
```

Comment:
Again, is this exclusively a sub-section of "things to keep in mind while testing"?